frontend/app/api/similarity/route.ts

```diff
@@ -1,27 +1,30 @@
-import { NextResponse } from 'next/server';
+import { NextResponse } from "next/server";
 
 export async function POST(request: Request) {
-  const { userInput } = await request.json();
-  try {
-    // Make a request to the Flask app
-    const response = await fetch(process.env.NEXT_PUBLIC_BACKEND_URL+'/api/similarity', {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify({ user_input: userInput }),
-    });
+	const { userInput } = await request.json();
+	try {
+		// Make a request to the Flask app
+		const response = await fetch("http://127.0.0.1:5000/api/similarity", {
```

Comment:
Replace the localhost url with what it was befre else it will break the prod of frontend